mbta_finder.py

```diff
@@ -35,8 +44,27 @@ def get_lat_long(place_name):
     See https://developers.google.com/maps/documentation/geocoding/
     for Google Maps Geocode API URL formatting requirements.
     """
-    pass
-
+    latitude = place_name["results"][0]["geometry"]["location"]["lat"]
+    longitude = place_name["results"][0]["geometry"]["location"]["lng"]
+    return latitude, longitude
+
+get_lat_long(response_data)
```

Comment:
Code not within a method (and similar testing code) should probably be located in `if __name__ == '__main__':`, which is run when the file is directly called.